.github/workflows/main.yml

```diff
@@ -30,6 +32,15 @@ jobs:
               flavor: centos
               major: 7
             java_major: 17
+          - base_image:
+              flavor: rockylinux
+              major: 9
+            java_major: 11
+          - base_image:
+              flavor: rockylinux
+              major: 9
+            java_major: 17
+            jdist: jdk
```

Comment:
I think we can remove jdk builds completely. AFAIK nobody has ever used them and we switched to JRE long ago